models/src/main/scala/coop/rchain/models/ParMapTypeMapper.scala

```diff
@@ -11,7 +11,10 @@ object ParMapTypeMapper {
 
   private[models] def parMapToEMap(parMap: ParMap): EMap =
     EMap(
-      parMap.ps.sortedList.map(t => zip(t._1, t._2)),
+      // Convert to Vector because with Scala 0.12.5 deserialized empty List (Nil)
```

Comment:
Did you mean `2.12.5`? It is similar to RNode version :slightly_smiling_face:

Reply:
Thanks! Yes, I meant Scala version 2.12.15. :joy: